### PR TITLE
Split SpriteMaskFilter into separate frag and vert files

### PR DESCRIPTION
**Pull request by woutercommandeur**
This resolves a TODO. It's the small things that matter I guess.

### CODE REVIEW

**Comment by englercj** [comment]
Lowercase the first letter of the frag/vert files, and fix the indentation to be 4 spaces in the shader code and I will merge it.

Thanks for this!

**Comment by woutercommandeur** [comment]
I can fix the indentation, however macos is case insensitive, and won't let me fix the first letter :(

**Comment by woutercommandeur** [comment]
I guess I can move to a completely different letter, commit, and move to lowercase first letter, but that would look silly in the commits

**Comment by GoodBoyDigital** [comment]
we won't judge, promise :)

**Comment by woutercommandeur** [comment]
That fixed it :)

File: src/core/renderers/webgl/filters/SpriteMaskFilter.js [hunk 1]
```diff
@@ -13,55 +13,9 @@ function SpriteMaskFilter(sprite)
...
-            '   float clip = 1.0 - max(text.y, text.x);',
-
-            '   vec4 original =  texture2D(uSampler, vTextureCoord);',
-            '   vec4 masky =  texture2D(mask, vMaskCoord);',
-            '   original *= (masky.r * masky.a * clip);',
-            '   gl_FragColor =  original;',
-            '}'
-        ].join('\n'),
-        // uniforms
+        fs.readFileSync(__dirname + '/SpriteMaskFilter.vert', 'utf8'),
```
Comment (englercj) [inline src/core/renderers/webgl/filters/SpriteMaskFilter.js]: These are still uppercased :) Needs to be lowered here in filename as well.

**Comment by woutercommandeur** [comment]
fixed, macos doesn't complain about that it just loads it.. so the errors don't show up :(

**Comment by englercj** [comment]
awesome dude, thanks for all the nitpicky changes I appreciate it!

**Pull request merged**